virttest/vt_utils/block_disk.py

```diff
@@ -0,0 +1,395 @@
+#
+# library for block related helper functions
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; specifically version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: Red Hat (c) 2023 and Avocado contributors
+# Author: Houqi Zuo <hzuo@redhat.com>
+import logging
+import platform
+import re
+
+from avocado.utils import process
+from avocado.utils import wait
+from virttest import utils_numeric
+
+PARTITION_TYPE_PRIMARY = "primary"
+SIZE_AVAILABLE = "available size"
+
+LOG = logging.getLogger('avocado.' + __name__)
+
+
+def is_mount(src, dst=None, fstype=None, options=None):
+    """
+    Check whether the src or dst is mounted or not.
+
+    :param src: source device or directory
+    :type src: string
+    :param dst: mountpoint, it will be replaced to ''( nothing ) if it's None
+    :type dst: string
+    :param fstype: filesystem type, it will be replaced to ''( nothing )
+                   if it's None
+    :type fstype: string
+    :param options: mount options, which should be separated by ","
+                    between each options
+    :type options: string
+
+    :return: True if mounted, else return False
+    :rtype: boolean
+    """
+    mount_str = "%s %s %s" % (src, dst, fstype)
+    mount_str = mount_str.replace('None', '').strip()
+    mount_list_cmd = 'cat /proc/mounts'
+
+    mount_result = process.run(mount_list_cmd, shell=True).stdout_text
+
+    for result in mount_result.splitlines():
+        if mount_str in result:
+            if options:
+                options = options.split(",")
+                options_result = result.split()[3].split(",")
+                for op in options:
+                    if op not in options_result:
+                        return False
+            return True
+    return False
+
+
+def mount(src, dst, fstype=None, options=None, verbose=False):
```

Comment:
```suggestion
def mount(src, dst, fstype=None, options=None):
```
Looks like it is useless.